scripts/azureml-assets/azureml/assets/validate_assets.py

```diff
@@ -470,17 +491,21 @@ def validate_model_assets(latest_asset_config: assets.AssetConfig, validated_ass
             if latest_model_config.type != validated_model_config.type:
                 logger.log_warning(f"latest_model_config_type: [{latest_model_config.type}]")
                 logger.log_warning(f"validated_model_config_type: [{validated_model_config.type}]")
+                error_count += 1
 
             if latest_model_config.path.type != validated_model_config.path.type:
                 logger.log_warning(f"latest_model_config_path_type: [{latest_model_config.path.type}]")
                 logger.log_warning(f"validated_model_config_path_type: [{validated_model_config.path.type}]")
+                error_count += 1
 
             if latest_model_path_uri != validated_model_path_uri:
                 logger.log_warning(f"latest_model_config_path_uri: [{latest_model_path_uri}]")
                 logger.log_warning(f"validated_model_config_path_uri: [{validated_model_path_uri}]")
+                error_count += 1
 
             if latest_model_config.description != validated_model_config.description:
                 logger.log_warning("Description does not match, for latest and validated asset")
+                error_count += 1
 
             return 1

```

Comment:
I think this line needs to be changed because this would disregard all the error_counts in this if-statement

Reply:
Thanks Kelly. Removed return stmt